app/assets/stylesheets/components/tooltips.scss

```diff
@@ -0,0 +1,31 @@
+.crayons-tooltip[data-tooltip] {
+  position: relative;
```

Comment:
Is it worth adding

```css
cursor: help;
```

Reply:
Good idea!

Reply:
Or actually... Since this can be applied to ANY element (like links, etc) I don't think we want to overwrite classic `cursor: pointer` with `help` one in these cases.